src/evergreen/metrics/buildmetrics.py

```diff
@@ -27,6 +27,12 @@ def __init__(self, build):
         self.timed_out_count = 0
         self.system_failure_count = 0
 
+        self.success_display_count = 0
```

Comment:
There must be a more accurate name for "display"

Reply:
Would it make sense to flip the order, `display_success_count`?